src/routes/authorsRoute.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const { createAuthor, findAllAuthors, findAuthorByPk, updateAuthor, deleteAuthor } = require('../controllers/authorsController');
```

Comment:
Much cleaner to write this as one import per line:
```suggestion
const express = require('express');
const {
    createAuthor,
    findAllAuthors,
    findAuthorByPk,
    updateAuthor,
    deleteAuthor,
} = require("../controllers/authorsController");
```